README.md

```diff
@@ -7,17 +7,26 @@ Internally uses:
 * [RxBluetoothKit](https://github.com/Polidea/RxBluetoothKit) 
  
 If you'd like to use it in your own library, you should start from BleAdapter on either platform.
+
+### Android setup
+
+Make sure to instruct the users of your library about the need to add and configure a global error handler in their application.
+https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling
```

Comment:
I think we should be more specific and link here as well
: https://github.com/Polidea/RxAndroidBle/wiki/FAQ:-UndeliverableException